text/0199-support-elastic-common-schema-in-opentelemetry.md

```diff
@@ -0,0 +1,297 @@
+# Support Elastic Common Schema in OpenTelemetry
+
+Collaborators: Alexander Wert (Elastic), Jamie Hynds (Elastic), Alolita Sharma (OTel GC, Apple), Christian Beedgen (Sumo), Jonah Kowall (Logz.io), Tigran Najaryan (Splunk).
```

Comment:
Is this list up to date?

Reply:
Or is this about the people involved in creating the proposal, rather than people involved/interested in making this effort happen?

Reply:
This is the list of people initially (when Cyrille created the first proposal) interested in the effort.